app/src/main/java/com/adsbynimbus/android/sample/Settings.kt

```diff
@@ -35,7 +34,7 @@ fun SharedPreferences.initNimbusFeatures(features: Set<String> = all.keys) {
                 }.apply()
             }
             "ccpa_consent" -> Nimbus.usPrivacyString = "1NYN".takeIf { _ -> getBoolean(it, false) }
-            "enable_viewability" -> ViewabilityProvider.thirdPartyViewabilityEnabled = getBoolean(it, true)
+            "enable_viewability" -> Nimbus.thirdPartyViewabilityEnabled = getBoolean(it, true)
```

Comment:
Just confirming, this change is not required, right? I'm assuming that you did it just to exemplify how to set it OFF.

Reply:
Correct.